setup/lib/utils.js

```diff
@@ -0,0 +1,74 @@
+const { setFailed } = require('@actions/core');
+const os = require('os');
+const path = require('path');
+const execSync = require('child_process').execSync;
+const semver = require('semver');
+const core = require('@actions/core');
+
+// arch in [arm, x32, x64...] (https://nodejs.org/api/os.html#os_os_arch)
+// return value in [amd64, 386, arm]
+function mapArch(arch) {
+    const mappings = {
+        x32: '386',
+        x64: 'amd64'
+    };
+    return mappings[arch] || arch;
+}
+
+// os in [darwin, linux, win32...] (https://nodejs.org/api/os.html#os_os_platform)
+// return value in [darwin, linux, windows]
+function mapOS(os) {
+    const mappings = {
+        darwin: 'macOS',
```

Comment:
Looks like we don't need to map the name for `darwin`: `notation_1.0.0-rc.7_darwin_amd64.tar.gz`

Reply:
Is this because the binary name uses the term `darwin`, instead of `macOS`? And because the `os.platform()` call returns `darwin` there's no need to map the values?

Reply:
Yes, you can find all the artifacts' name here: https://github.com/notaryproject/notation/releases/tag/v1.0.0-rc.7